archive/archive_api/src/apis/bags.py

```diff
@@ -32,22 +32,17 @@ def get(self, space, id):
         """Get the bag associated with an id"""
         from archive_api import app
 
-        bag_id = f"{space}/{id}"
+        bags_manager = app.config["BAGS_MANAGER"]
 
         try:
-            result = read_from_vhs(
-                dynamodb_resource=app.config["DYNAMODB_RESOURCE"],
-                table_name=app.config["BAG_VHS_TABLE_NAME"],
-                s3_client=app.config["S3_CLIENT"],
-                bucket_name=app.config["BAG_VHS_BUCKET_NAME"],
-                id=bag_id,
-            )
+            result = bags_manager.lookup_bag(space, id)
 
             # TODO: Remove the necessity to do this
-            result["id"] = result["id"]["value"]
+            result[
+                "id"
+            ] = f'{result["id"]["space"]}/{result["id"]["externalIdentifier"]}'
```

Comment:
I think I might just turn off black if this is the formatting it's going to apply, because this is bizarre.

Reply:
It is, but it's probably my fault for putting too much into the same line. I don't think it can break that line differently?